src/signals/incident/services/custom-validators/index.js

```diff
@@ -44,3 +50,8 @@ export const validatePhoneNumber = control => {
 
   return { custom: 'Ongeldig telefoonnummer, alleen cijfers, spaties, haakjes, + en - zijn toegestaan.' };
 };
+
+export const validateRequired = message =>
+  function customRequired({ value }) {
+    return value === null || value === undefined || value.length === 0 ? { custom: message } : null;
```

Comment:
if the returnn would be `{required: message}`, this will be used in the required validator as error message

Reply:
😃 nice one! Makes it much better